.github/workflows/ci_dockers.yml

```
@@ -73,9 +73,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # the config used in '.azure-pipelines/gpu-tests.yml'
-        python_version: ["3.7"]
-        pytorch_version: ["1.8"]
+        include:
+          # the config used in '.azure-pipelines/gpu-tests.yml'
+          - {python_version: "3.7", pytorch_version: "1.8"}
+          - {python_version: "3.9", pytorch_version: "1.10"}
```

Two questions:
- Where will (py3.9, pt1.10) be used?
- Do you think it's reasonable to test (py3.9, pt1.11) instead of (py3.9, pt1.10) here?

```suggestion
          - {python_version: "3.9", pytorch_version: "1.11"}
```

---

it is used in Tutorials:
https://github.com/PyTorchLightning/lightning-tutorials/blob/0bb4a7e4ba5053a36c9bf229447f5332c9d848f4/.azure/ipynb-publish.yml#L24
and I may say that 1.10 is more stable then the recent 1.11 (lets update later)